studio/components/interfaces/Settings/Vault/Secrets/SecretRow.tsx

```diff
@@ -107,20 +110,23 @@ const SecretRow = ({ secret, onSelectEdit, onSelectRemove }: SecretRowProps) =>
           {secret.updated_at === secret.created_at ? 'Added' : 'Updated'} on{' '}
           {dayjs(secret.updated_at).format('MMM D, YYYY')}
         </p>
-        <Dropdown
-          side="bottom"
-          className="w-[120px]"
-          overlay={
+        <DropdownMenu_Shadcn_>
+          <DropdownMenuTrigger_Shadcn_>
+            <Button asChild type="text" className="px-1" icon={<IconMoreVertical />}>
+              <span></span>
+            </Button>
+          </DropdownMenuTrigger_Shadcn_>
+          <DropdownMenuContent_Shadcn_ side="bottom" className="w-[120px]">
             <>
               <Tooltip.Root delayDuration={0}>
```

Comment:
This tooltip messes up the dropdown. Probably have to be migrated to shadcn one.